WalletWasabi/Helpers/WindowsStartupChecker.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.Win32;
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace WalletWasabi.Helpers
+{
+	public static class WindowsStartupChecker
+	{
+		private const string KeyPath = "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run";
+
+		public static bool CheckRegistryKeyExists()
```

Comment:
That sanity-check is only for eliminating a warning below, where we use the `RegistryKey` class. I could surpress the warning, but i find this solution cleaner.

Reply:
IMO it's certainly not clean. There are two approaches that makes sense to me:

1) You can use a suppression and your code is shorter and if it is used on an unsupported platform, it will crash or whatever (which is fine because there is a bug in code)
2) You can use `if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))` but then on unsupported platforms, you should throw some *not supported exception*.

Reply:
I will go with the first suggestion.